### ISSUE TITLE
[RETEST] Mobile A11Y: PDP/Homepage Carousel buttons are hard to reach and not labelled.

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13757]
**Additional context**

- Fix this ticket only after the whole Screen Reader milestone for desktop is completed. 

**Describe the bug**

- PDP/Homepage Carousel buttons are hard to reach and not labelled.

**To Reproduce: Watch Test Videos**

- Android Talkback test video
https://video.sap.com/media/t/1_1gtfpn6o
min 0:07:00
min 1:53:23
  
- Iphone VoiceOver test video
https://video.sap.com/media/t/1_qa70rxtl
min 0:03:28

**Expected behavior**
- Carousel buttons have an accessible pattern and get vocalized. Eg. _"Previous slide", "Next slide", "Slide 1", "Slide 2", etc._
